webaverse-render-pass.js

```diff
@@ -28,24 +29,49 @@ class WebaverseRenderPass extends Pass {
       this.internalRenderPass.setSize(width,height);
     }
 
-	}
+  }
   render(renderer, renderTarget, readBuffer, deltaTime, maskActive) {
     this.onBeforeRender && this.onBeforeRender();
-    
-    // render
-    if (this.internalDepthPass) {
-      this.internalDepthPass.renderToScreen = false;
-      this.internalDepthPass.render(renderer, renderTarget, readBuffer, deltaTime, maskActive);
-    }
-    if (this.internalRenderPass) {
-      this.internalRenderPass.renderToScreen = this.renderToScreen;
-      this.internalRenderPass.render(renderer, renderTarget, readBuffer, deltaTime, maskActive);
-    } else {
-      renderer.setRenderTarget(renderTarget);
-      renderer.clear();
-      renderer.render(rootScene, camera);
+
+    if(getRenderer().xr.getSession()) {
+      // render
+      if (this.internalDepthPass) {
+        this.internalDepthPass.renderToScreen = false;
+        this.internalDepthPass.render(renderer, renderTarget, readBuffer, deltaTime, maskActive);
+      }
+      if (this.internalRenderPass) {
+        this.internalRenderPass.renderToScreen = this.renderToScreen;
+        this.internalRenderPass.render(renderer, renderTarget, readBuffer, deltaTime, maskActive);
+      } else {
+        
+        // renderer.clear();
+        // renderer.render(scene, camera);
+        // renderer.clear();
+        // renderer.render(sceneHighPriority, camera);
+        // renderer.clear();
+        // renderer.render(sceneLowPriority, camera);
+        //console.log(renderTarget);
+        renderer.clear();
+        renderer.setRenderTarget( renderer.getRenderTarget() );

```

Comment:
Is this completely bypassing the postprocessing? This seems the wrong place to hook VR. It should be dowstream of this.